### PR TITLE
Feat : Login and Registry

### PR DESCRIPTION
**Pull request by guarneriAnthony**
Can you validate the pull request please scrum Master ?

### CODE REVIEW

File: Back-end/src/main/java/com/lacorp/backend/model/User.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
+package com.lacorp.backend.model;
```
Comment (Mekouar-Ayoub) [inline Back-end/src/main/java/com/lacorp/backend/model/User.java]: Appellez le UserRepositoryModel

File: Back-end/src/main/java/com/lacorp/backend/service/impl/JwtUserServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    public JwtUserServiceImpl(@Value("${jwt.signing.key}") String signingKey) {
+        this.signingKey = signingKey;
+    }
+    @Override
+    public UserDetails save(String username, String password, String email) throws AccountExistsException {
+        UserDetails existingUser = userRepository.findByUsername(username);
+        if (existingUser != null){
+            throw new AccountExistsException();
+        }
+        User user = new User();
```
Comment (Mekouar-Ayoub) [inline Back-end/src/main/java/com/lacorp/backend/service/impl/JwtUserServiceImpl.java]: Utilise un Constructeur plutot

**Review by Mekouar-Ayoub — CHANGES_REQUESTED** [review]

File: Back-end/src/main/java/com/lacorp/backend/model/User.java (hunk 1 above)
Comment (guarneriAnthony) [reply Back-end/src/main/java/com/lacorp/backend/model/User.java]: Done

File: Back-end/src/main/java/com/lacorp/backend/service/impl/JwtUserServiceImpl.java (hunk 2 above)
Comment (guarneriAnthony) [reply Back-end/src/main/java/com/lacorp/backend/service/impl/JwtUserServiceImpl.java]: Done

**Pull request merged**